packages/synapse-react-client/src/components/DraggableDialog/DraggableDialog.tsx

```diff
@@ -115,7 +115,15 @@ export default function DraggableDialog({
   )
 
   return (
-    <Box sx={{ position: 'fixed', zIndex: 1000, top: 0, left: 0 }}>
+    <Box
+      sx={{
+        position: 'fixed',
+        zIndex: 1000,
+        top: 0,
+        left: 0,
+        pointerEvents: isMobile ? 'auto' : 'none',
```

Comment:
Allow pointer events through this fixed container, but continue to capture pointer events on the transformed/translated element